doc/security.md

```diff
@@ -1,289 +1,129 @@
 # Security
-## Introduction
-LXD is a daemon running as root.
 
-Access to that daemon is only possible over a local UNIX socket by default.
-Through configuration, it's then possible to expose the same API over
-the network on a TLS socket.
+% Include content from [../README.md](../README.md)
+```{include} ../README.md
+    :start-after: <!-- Include start security -->
+    :end-before: <!-- Include end security -->
+```
+
+See the following sections for detailed information.
 
-**WARNING**: Local access to LXD through the UNIX socket always grants
```

Comment:
So we'll definitely want some version of this warning to be prominently visible on this page. Re-introducing it in the main README.md as already mentioned will hopefully do the trick through the include.